server/game/cards/03-WotN/YoungSpearwife.js

```diff
@@ -10,11 +10,8 @@ class YoungSpearwife extends DrawCard {
     }
 
     hasLessFactionPower() {
-        var otherPlayer = this.game.getOtherPlayer(this.controller);
-        if(!otherPlayer) {
-            return false;
-        }
-        return this.controller.faction.power < otherPlayer.faction.power;
+        let opponents = this.game.getOpponents(this.controller);
+        return opponents.some(opponent => this.controller.faction.power < opponent.faction.power);
```

Comment:
who on earth decided 'some' was better than 'any'? :shakes_head_at_es6:

Reply:
Still waffling about whether to introduce aliases for `any` and `all`. Trying to stick w/ standard JS for now but I agree, the standard naming is terrible.